### DISCUSSION TITLE
Slider followcircle bug in editor

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#20128] [Q&A] [answered]
Not sure how to explain it properly. 

https://user-images.githubusercontent.com/53872073/188335235-f13fe7cd-3c6a-406a-9cfb-d00bedb3e919.mp4

Steps:
1) Create a beatmap / start editing one.
2) Place a very long slider.
3) Be in it's middle.
4) Change beatmap's CS on "setup" tab.
5) Return to compose tab, try to seek a little
6) Notice that followcircle disappears at moment where you was when changing CS.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20132
